dirty_cat/super_vectorizer.py

```diff
@@ -95,11 +126,11 @@ class SuperVectorizer(ColumnTransformer):
         data type (dtype).
 
     impute_missing : str, default='auto'
-        When to impute missing values in string columns.
+        When to impute missing values in categorical columns.
```

Comment:
Shouldn't it be "string" rather than categorical? This is applied to string columns, no? The notion of categorical is slightly different (there is for instance the notion of categorical dtype in pandas, that is not string)

Reply:
Well, it's for both. Check the content of `_replace_missing_in_cat_col` to see why.
I renamed the method this way (it was `_replace_missing_in_col` before) to clarify that we're not imputing on numerical columns.

Reply:
To my surprise the code works on categorical dtype (which tells me that my understanding of categorical dtype is limited).

Still, it seems important to me that the docstring says "string and categorical columns", because it works on strings too, and the "dirty categories" that dirty-cat focuses on are not considered categorical by everybody.